README.md

```diff
@@ -166,10 +167,19 @@ See [FactorListPresenter](https://github.com/twilio/twilio-verify-ios/blob/main/
 
 ## Delete a factor
 You can delete a factor calling the `TwilioVerify.deleteFactor` method:
-```
+```swift
 twilioVerify.deleteFactor(withSid: factorSid, success: {
   // Success
 }) { error in
   // Error
 }
 ```
+
+<a name='ClearLocalStorage'></a>
+
+## Clear local storage
+You can clear the local storage calling the `TwilioVerify.clearLocalStorage` method:
+```swift
+twilioVerify.clearLocalStorage()
```

Comment:
Should we add `try`?

Reply:
Can we have a method that doesn't throw an exception?

Reply:
I thought about it and we can do it but it feel incomplete to me, I know that if it fails theres no much the consumer can do but I think we should at least give them the option.

PS I will add the `try` to the readme depending on what we decide about this

Reply:
Which errors could happen clearing the storage?

Reply:
Keychain errors

Reply:
Could you please investigate the possible errors?

Reply:
I would like to know if an error could prevent the factors deletion. We are using the same method to delete one item, for one item an error could be item not found, but for this case we are deleting all items, so I am not sure the errors we can see

Reply:
The errors are the same we can have when deleting a single factor

Reply:
Can be an error thrown clearing local storage but adding a factor can work?

Reply:
It can, I added a note on the README explaining why it could happen